src/main/thrift/parquet.thrift

```diff
@@ -626,11 +628,9 @@ struct FileMetaData {
   6: optional string created_by
 
   /**
-   * Sort order used for each column in this file.
-   *
-   * If this list is not present, then the order for each column is assumed to
```

Comment:
I removed this sentence since it no longer applied. (It was a leftover from the early stages of PR #46 when this list was meant to modify the behavior of the `min` and `max` fields, but they were made obsolete instead.) However, this also means that the behavior for a missing list is undefined.

@rdblue @julienledem How should we define the meaning of `min_value` and `max_value` when this list is not present? @lekv What does Impala do if this list is not present?